attestation-container/attestation-container_test.go

```diff
@@ -5,31 +5,36 @@ import (
 	"encoding/hex"
 	"flag"
 	"log"
+	"net"
 	"testing"
 	"time"
 
 	pb "microsoft/attestation-container/protobuf"
 
 	"google.golang.org/grpc"
-	"google.golang.org/grpc/credentials/insecure"
 )
 
 var (
-	addr = flag.String("addr", "localhost:50051", "the address to connect to")
+	addr = flag.String("addr", "/tmp/attestation-container.sock", "the Unix domain socket address to connect to")
 )
 
+const GPRC_TIMEOUT_IN_SEC = 10
```

Comment:
This is not related to UDS itself, but I faces 'not responded' error in the CI several times. It's the fix for it and looks like it's working.

Reply:
Same fix was made in https://github.com/microsoft/CCF/pull/4858